.codeclimate.yml

```diff
@@ -1,5 +1,44 @@
+version: "2"
+
 languages:
   Python: true
+  Javascript: true
+
+checks:
+  argument-count:
+    enabled: true
+    config:
+      threshold: 6
+  complex-logic:
+    enabled: true
+    config:
+      threshold: 8
+  file-lines:
+    enabled: true
+    config:
+      threshold: 250
+  method-complexity:
+    enabled: true
+    config:
+      threshold: 15
+  method-count:
+    enabled: true
+    config:
+      threshold: 20
+  method-lines:
+    enabled: true
+    config:
+      threshold: 45
+  nested-control-flow:
+    enabled: true
+    config:
+      threshold: 5
+  return-statements:
+    enabled: true
+    config:
+      threshold: 3
```

Comment:
may want to revisit this one. sometimes it's better to short circuit the function with a return statement rather than adding additional control flow that just continues to the bottom of the function. May need to be a case by case basis for this.

Reply:
I agree that sometimes it is better/cleaner to "short-circuit" a function. My thought process is that if the logic requires 4 or more return statements perhaps the function should be refactored.